test/e2e/framework/pv_util.go

```diff
@@ -621,13 +623,28 @@ func MakePersistentVolumeClaim(cfg PersistentVolumeClaimConfig, ns string) *v1.P
 	// Specs are expected to match this test's PersistentVolume
 
 	if len(cfg.AccessModes) == 0 {
-		e2elog.Logf("AccessModes unspecified, default: all modes (RWO, RWX, ROX).")
+		e2elog.Logf("Making PVC: AccessModes unspecified, default: all modes (RWO, RWX, ROX).")
 		cfg.AccessModes = append(cfg.AccessModes, v1.ReadWriteOnce, v1.ReadOnlyMany, v1.ReadOnlyMany)
 	}
 
+	if len(cfg.ClaimSize) == 0 {
+		e2elog.Logf("Making PVC: ClaimSize unspecified, default: 2Gi")
+		cfg.ClaimSize = "2Gi"
+	}
+
+	if len(cfg.NamePrefix) == 0 {
+		e2elog.Logf("Making PVC: NamePrefix unspecified, default: pvc-")
+		cfg.NamePrefix = "pvc-"
+	}
+
+	if cfg.VolumeMode != nil && *cfg.VolumeMode == "" {
+		e2elog.Logf("Warning: Making PVC: VolumeMode specified as invalid empty string, treating as nil")
```

Comment:
I know this isn't ideal but I wanted to keep the refactor small-ish. The issue is actually in the callers not using a pointer to the `VolumeMode` throughout the test code so they're defaulting to empty string sometimes. Would be a much larger refactor to fix - we can just treat them as nil here (to default) and log a warning

Reply:
fyi an empty string `VolumeMode` is actually an invalid value

Reply:
Yeah, a fix for another time